script/download-node.js

```diff
@@ -110,7 +115,8 @@ var downloadNode = function(version, done) {
   }
 };
 
-downloadNode('v0.10.35', function(error) {
+var apmNodeVersion = process.env.APM_NODE_VERSION || '0.10.35';
```

Comment:
@zcbenz I'm not sure - I ran into https://github.com/atom/apm/issues/410 by building with an older (0.10.x) of Node.